README-ZH.md

```diff
@@ -50,21 +50,23 @@ https://github.com/UnityTech/DocCN 查看。
 
 #### Unity
 
-安装 **Unity 2018.4.10f1(LTS)** 或 **Unity 2019.1.14f1** 及其更高版本。 你可以从[https://unity3d.com/get-unity/download](https://unity3d.com/get-unity/download)下载最新的Unity。
+安装 **Unity 2019.1.14f1c1** 及其更高版本。 你可以从[https://unity3d.com/get-unity/download](https://unity3d.com/get-unity/download)下载最新的Unity。
 
 #### UIWidgets包
 
-访问我们的Github存储库 [https://github.com/UnityTech/UIWidgets](https://github.com/UnityTech/UIWidgets)下载最新的UIWidgets包。
+访问我们的Github存储库 [https://github.com/Unity-Technologies/com.unity.uiwidgets](https://github.com/Unity-Technologies/com.unity.uiwidgets)下载最新的UIWidgets包。
 
 将下载的包文件夹移动到Unity项目的Package文件夹中。
 
 通常，你可以在控制台（或终端）应用程序中输入下面的代码来完成这个操作：
 
    ```none
     cd <YourProjectPath>/Packages
-    git clone https://github.com/UnityTech/UIWidgets.git com.unity.uiwidgets
+    git clone https://github.com/Unity-Technologies/com.unity.uiwidgets.git com.unity.uiwidgets
```

Comment:
user should clone uiwdigets to **<YourProjectPath>/com.unity.uiwidgets** and add local file link in **Packages/manifest.json** to resolve com.unity.uiwidgets to **<YourProjectPath>/com.unity.uiwidgets/com.unity.uiwidgets**

The reason is that, the current package structure works only for package manager